PFR/ForMathlib/Entropy/RuzsaDist.lean

```diff
@@ -177,19 +183,20 @@ lemma rdist_of_inj  {H:Type*} [hH : MeasurableSpace H] [MeasurableSingletonClass
       have : φ ∘ (fun x ↦ x.1 - x.2) = g ∘ f := by
         ext x
         simp
-      rw [this, ← MeasureTheory.Measure.map_map (g := g) (f := f) (measurable_of_countable _) (measurable_of_countable _), ← MeasureTheory.Measure.map_map (measurable_of_countable _) hX, ← MeasureTheory.Measure.map_map (measurable_of_countable _) hY]
+      rw [this, ← MeasureTheory.Measure.map_map (g := g) (f := f) (measurable_of_countable _)
+        (measurable_of_countable _), ← MeasureTheory.Measure.map_map (measurable_of_countable _) hX,
+        ← MeasureTheory.Measure.map_map (measurable_of_countable _) hY]
       congr
-      convert MeasureTheory.Measure.map_prod_map _ _ (measurable_of_countable _) (measurable_of_countable _)
+      convert MeasureTheory.Measure.map_prod_map _ _ (measurable_of_countable _)
+        (measurable_of_countable _)
```

Comment:
Maybe you could introduce `lemma Measurable.of_countable {f : α → β} := measurable_of_countable _` (put it in `PFR.Mathlib.MeasureTheory.MeasurableSpace.Basic`) so that this can become
```suggestion
      convert Measure.map_prod_map _ _ .of_countable .of_countable
```

Reply:
Sure, thanks a lot. I'm going to do it later today or tomorrow.

Reply:
Pushed the change right now: 

https://github.com/pitmonticone/Polynomial_Freiman-Ruzsa_Conjecture/blob/c8059aa7b032605a17f940034560455518aa3321/PFR/ForMathlib/Entropy/RuzsaDist.lean#L190